guides/common/modules/proc_enable-insights-with-capsule.adoc

```diff
@@ -0,0 +1,25 @@
+[id='register-capsule-with-insights_{context}']
+
+= Register :SmartProxyServer: with Red Hat Insights

```

Comment:
You need {SmartProxyServer} for the variable to work. 
Please follow mod docs/IBM style standards for topic headings.